### PR TITLE
Added constraints to the "List messages" params.

### PR DESCRIPTION
**Pull request by amatkivskiy**
Hi guys,

Applied @MadLittleMods suggestions about creating PR regarding the [List messages parameters](https://github.com/gitterHQ/docs/blob/master/04.Messages-resource.md#parameters).

### CODE REVIEW

File: 04.Messages-resource.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ List of messages in a room.
 
 All the parameters are optional:
 
- - `skip`: Skip n messages.
+ - `skip`: Skip n messages (constrained to 5000 or less).
  - `beforeId`: Get messages before `beforeId`.
  - `afterId`: Get messages after `afterId`.
- - `aroundId`: Get messages around `aroundId` including this message.
+ - `aroundId`: Get messages around `aroundId` including this message (constrained to 100 or less).
```
Comment (MadLittleMods) [inline 04.Messages-resource.md]: This should be on the `limit` parameter :grinning:

Comment (amatkivskiy) [inline 04.Messages-resource.md]: Sorry, seems like I was drunk. :laughing:

**Comment by amatkivskiy** [comment]
@MadLittleMods fixed :smile: .

**Comment by MadLittleMods** [comment]
@amatkivskiy Thank you for updating that part of the docs :grinning:

**Pull request merged**